### PR TITLE
8294519: (fs) java/nio/file/Files/CopyProcFile.java fails intermittenly due to unstable /proc/cpuinfo output

### PR DESCRIPTION
**Pull request by bplb**
1. Change source from `/proc/cpuinfo` to `/proc/version`.
2. Do not swallow errors in `@BeforeTest` config action.
3. Remove test from problem list.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8294519](https://bugs.openjdk.org/browse/JDK-8294519): (fs) java/nio/file/Files/CopyProcFile.java fails intermittenly due to unstable /proc/cpuinfo output

### Reviewers
 * [Alan Bateman](https://openjdk.org/census#alanb) (@AlanBateman - **Reviewer**) ⚠️ Review applies to [c80329e6](https://git.openjdk.org/jdk/pull/10479/files/c80329e63d800a6a78ca7e8dde03118ffb1b0a34)
 * [Aleksey Shipilev](https://openjdk.org/census#shade) (@shipilev - **Reviewer**) ⚠️ Review applies to [c80329e6](https://git.openjdk.org/jdk/pull/10479/files/c80329e63d800a6a78ca7e8dde03118ffb1b0a34)
 * [Lance Andersen](https://openjdk.org/census#lancea) (@LanceAndersen - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/10479/head:pull/10479` \
`$ git checkout pull/10479`

Update a local copy of the PR: \
`$ git checkout pull/10479` \
`$ git pull https://git.openjdk.org/jdk pull/10479/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10479`

View PR using the GUI difftool: \
`$ git pr show -t 10479`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/10479.diff">https://git.openjdk.org/jdk/pull/10479.diff</a>

</details>

### CODE REVIEW

**Review by AlanBateman — APPROVED** [review]
file /proc/version reports that it is empty but you can read content, so I think it works to exercise the code paths.

File: test/jdk/java/nio/file/Files/CopyProcFile.java [hunk 1]
```diff
@@ -115,18 +113,19 @@ static long transferFrom(String src, String dst) {
...
-            if (Files.mismatch(Path.of(BUFFERED_COPY), Path.of(SOURCE)) != -1)
-                throw new RuntimeException("Copy does not match source");
-        } catch (Exception e) {
+        } catch (IOException e) {
             try {
                 Files.delete(Path.of(BUFFERED_COPY));
             } catch (IOException ignore) {}
+            throw e;
         }
+        if (Files.mismatch(Path.of(BUFFERED_COPY), Path.of(SOURCE)) != -1)
```
Comment (shipilev) [inline test/jdk/java/nio/file/Files/CopyProcFile.java:127]: Braces?

**Review by shipilev — APPROVED** [review]
Looks okay to me, and test passes on the system where it failed before.

**Review by LanceAndersen — APPROVED** [review]

**Review by LanceAndersen — APPROVED** [review]

**Pull request closed without merging**